src/Workspaces/Core/Portable/FindSymbols/SymbolFinder_FindReferences_Legacy.cs

```diff
@@ -82,7 +82,7 @@ public static partial class SymbolFinder
                 FindReferencesSearchOptions.Default, cancellationToken).ConfigureAwait(false);
         }
 
-        private static async Task<ImmutableArray<ReferencedSymbol>> FindReferencesAsync(
+        internal static async Task<ImmutableArray<ReferencedSymbol>> FindReferencesAsync(
```

Comment:
❔ Why did this need to change?

Reply:
CodeLens now calls into an entrypoint that allows for the options to be passed in.  Effectively, before this it called the overload that then called into this (without options provided).  Now, we call into thsi directly so we can pass the explicit options saying we are implicitly invoked.

Reply:
> If it's not for test purposes, does it need to be moved out of _Legacy?

Yeah... 'Legacy' effectively means: not using the new modern streaming api that reports matches when found, but isntead using the API that returns all the values in one array.  It is still used by parts of our system that haven't updated to the new APIs.

Reply:
But, importantly, i would like to keep it in the Legacy folder to strongly indicate: please move off of this when you can.

Reply:
works for me